mantidproject/mantid
# Add Comment Algorithm

This is for annotating workflows - adding comments into the generated python

closes #13867 
## Release Notes

http://www.mantidproject.org/ReleaseNotes_3_6_Framework_Changes#New
## To test
1. Create a workspace
2. Add a few comments within a few other algorithms
3. Check how they appear when python scripts are generated from them.
